ipyleaflet/leaflet.py

```diff
@@ -571,16 +571,16 @@ def _update_data(self, change):
     def _get_data(self):
         if self.style_callback:
             if self.data['type'] == 'Feature':
-                self.data['properties']['style'] = self.style_callback(self.data)
+                self.data['properties']['style'].update(self.style_callback(self.data))
             elif self.data['type'] == 'FeatureCollection':
                 for feature in self.data['features']:
-                    feature['properties']['style'] = self.style_callback(feature)
-        else:
+                    feature['properties']['style'].update(self.style_callback(feature))
+        elif self.style:
             if self.data['type'] == 'Feature':
-                self.data['properties']['style'] = self.style
+                self.data['properties']['style'].update(self.style)
             elif self.data['type'] == 'FeatureCollection':
                 for feature in self.data['features']:
-                    feature['properties']['style'] = self.style
+                    feature['properties']['style'].update(self.style)
```

Comment:
There should be a minor change here, as this code always expects that there will be `style` in properties of `geojson`. If `style` is not present in the properties this fails with `KeyError` exception. I think there should be one more check if `style` key is already present then update, else assign the value.
e.g. - 
```python
if 'style' in  feature['properties']:
   feature['properties']['style'].update(self.style)
else:
   feature['properties']['style'] = self.style
```

Reply:
for all update statements, need to check if `style` key is already present or not.

Reply:
I have raised PR #572